Source/WTF/wtf/SuspendableWorkQueue.h

```diff
@@ -36,7 +36,9 @@ namespace WTF {
 class SuspendableWorkQueue final : public WorkQueue {
 public:
     using QOS = WorkQueue::QOS;
-    WTF_EXPORT_PRIVATE static Ref<SuspendableWorkQueue> create(const char* name, QOS = QOS::Default);
+    enum class State : uint8_t { Running, WillSuspend, Suspended };
```

Comment:
Can we put it back where it was? Not sure why this needs to be public now.

Reply:
Because `static inline const char* stateString(SuspendableWorkQueue::State state)` in SuspendableWorkQueue.cpp?
Or do you mean I should change `stateString()` back to a class member?

Reply:
Oh I see. Yes, given this I'd prefer if we put the static function declaration in the header as private (so State can stay private). Just mark it as inline in the cpp.